### PR TITLE
Add support for gyro_cal_on_first_arm

### LINKED ISSUE #102: Add gyro_cal_on_first_arm
Currently only available in the CLI.  Probably should go on the Configuration tab but I'm not sure where.

### PR DESCRIPTION
**Pull request by haslinghuis**
- Fixes #102 
- Firmware PR https://github.com/betaflight/betaflight/pull/13626
- Support for disarm_kill_switch was removed in https://github.com/betaflight/betaflight/pull/5073
image

### CODE REVIEW

**Comment by haslinghuis** [comment]
Editor is happy to add vite imports all the time

image

File: locales/en/messages.json [hunk 1]
```diff
@@ -1395,13 +1395,13 @@
...
     },
-    "configurationDisarmKillSwitch": {
-        "message": "Disarm motors regardless of throttle value (When ARM is configured in Modes tab via AUX channel)"
+    "configurationAutoDisarmDelayHelp": {
+        "message": "Disarm motors after set delay [seconds] (Requires MOTOR_STOP feature)"
     },
-    "configurationDisarmKillSwitchHelp": {
-        "message": "Arming is always disabled when the throttle is not low. Be careful as you could disarm accidentally with a switch while flying with this option active."
+    "configurationGyroCalOnFirstArm": {
+        "message": "Calibrate Gyro on first arm"
```
Comment (KarateBrot) [inline locales/en/messages.json:1404]: This change maybe gives a slightly better hint as to what is happening (= gyro gets calibrated before motors start to spin)
```suggestion
        "message": "Calibrate the gyro right before the first arm"
```

**Review by KarateBrot — CHANGES_REQUESTED** [review]

**Review by KarateBrot — APPROVED** [review]

**Comment by nerdCopter** [comment]
my results are different, but this GPS enabled target has no actual GPS for testing.
image

**Comment by haslinghuis** [comment]
@nerdCopter Disarm motors after delay requires `MOTOR_STOP` feature (set in motor tab).

**Comment by nerdCopter** [comment]
> @nerdCopter Disarm motors after delay requires `MOTOR_STOP` feature (set in motor tab).

probably also requires ESC and motors :laughing:  
_but my screenshot also shown no toggle switch on the GPS-Calibrate option_.

**Comment by haslinghuis** [comment]
Yes checking here. Most have changed on rebase...

**Comment by nerdCopter** [comment]
the above comment was
```
# version
# Betaflight / STM32F405 (S405) 4.6.0 May 21 2024 / 08:20:12 (32f1e9a17) MSP API: 1.46
# config rev: 93dfc69
# board: manufacturer_id: FOXE, board_name: FOXEERF405
```
did not test `#13626`

**Pull request merged**